INTITA/Modules/TasksScreen/TasksViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  TasksViewController.swift
+//  INTITA
+//
+//  Created by Svitlana Korostelova on 25.12.2020.
+//
+
+import UIKit
+
+class TasksViewController: UIViewController, Storyboarded, AlertAcceptable, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UIScrollViewDelegate {
+    
+    var viewModel: TasksViewModel!
+    weak var coordinator: TasksCoordinator?
+    @IBOutlet weak var tasksHeadersCollectionView: UICollectionView!
+    @IBOutlet weak var taskTableView: UITableView!
+    @IBOutlet weak var priorities: UIButton!
+    @IBOutlet weak var types: UIButton!
+    @IBOutlet weak var groups: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.setNavigationBarHidden(false, animated: true)
+        let backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: self, action: #selector(backButtonPressed))
+                self.navigationController?.navigationBar.backItem?.backBarButtonItem = backBarButtonItem
+        //TODO:  set title for controller
+        let headerCell = UINib(nibName: "TaskHeaderCell", bundle: nil)
+        tasksHeadersCollectionView.register(headerCell, forCellWithReuseIdentifier: "HeaderReuseIdentifier")
+        let taskCell = UINib(nibName: "TaskCell", bundle: nil)
+        taskTableView.register(taskCell, forCellReuseIdentifier: "reuseForTask")
```

Comment:
You may use TaskHeaderCell.nib() and TaskHeaderCell.identifire, TaskCell.nib() and TaskCell.identifire to avoid hardcoded strings.

Reply:
done